lib/v2/screens/transfer/send/send_confirmation/interactor/mappers/send_transaction_state_mapper.dart

```diff
@@ -16,48 +14,28 @@ class SendTransactionStateMapper extends StateMapper {
       return currentState.copyWith(pageState: PageState.failure, errorMessage: result.asError!.error.toString());
     } else {
       final resultResponse = result.asValue!.value as SendTransactionResponse;
-      final transactionId = resultResponse.transactionId.asValue!.value as String;
 
-      final String quantity = currentState.data['quantity'].toString();
-      final currency = quantity.split(' ')[0];
-      final double parsedQuantity = double.parse(quantity.split(' ')[0]);
-
-      final selectedFiat = settingsStorage.selectedFiatCurrency;
-      final String fiatAmount = rateState.fromSeedsToFiat(parsedQuantity, selectedFiat).fiatFormatted;
-
-      if (areAllResultsSuccess(resultResponse.profiles)) {
-        final toAccount = resultResponse.profiles[0].asValue!.value as ProfileModel;
-        final fromAccount = resultResponse.profiles[1].asValue!.value as ProfileModel;
```

Comment:
this success dialog would only work for transfer since it made many assumptions about transfer-specific things, amount, fiat rate, etc.